app/form_objects/chromebook_information_form.rb

```diff
@@ -36,4 +37,25 @@ def recovery_email_address_cannot_be_same_domain_as_school_or_rb
       errors.add(:recovery_email_address, :cannot_be_same_domain_as_school_or_rb)
     end
   end
+
+  def will_need_chromebooks_and_is_not_a_la_funded_school?
+    if will_need_chromebooks?
+      if school.respond_to?(:la_funded_place?)
+        !school.la_funded_place?
+      else
+        true
+      end
```

Comment:
```suggestion
     !school.la_funded_place?
```

Do we still need to check if school responds to this method?

Reply:
Good call, we don't need this now, I've remove it.